### PR TITLE
Create modules to export in index.js file

### PR DESCRIPTION
**Pull request by yusufnviiri**
-Create folder modules.
-Create JavaScript modules using ES6 import and export functions
-Add luxon library
-Create index.HTML and  index.CSS files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
+-open style.css to see the CSS styles used in the project.
+-enjoy
+
+## 👤Author 
+### 👤 Yusuf Nviiri 
+
+-GitHub: [@yusufnviiri](https://github.com/yusufnviiri)
+-Twitter: [@YNviiri](https://twitter.com/YNviiri)
+-LinkedIn: [yusufnviiri]( https://www.linkedin.com/in/yusuf-nviiri-8b4146206/)
+
```
Comment (Mihndim2020) [inline README.md]: - [ ] Please your `Readme file` will be more presentable and professional if you place each point on it's own line. Some like so: 
- cd awesome books
- open index.html in the editor/browser to view the project contents
- open style.css to see the CSS styles used in the project.
- enjoy
To make each point appear on it's own line, leave a single space between the stroke and the first character on the line. Please kindly fix that.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
 Creating BookApp with   modular JavaScript
+
+## Built with
+
+-HTML
+-CSS
+-JavaScript
+
+## Live Demo
+ https://yusufnviiri.github.io/bookApp-ES6/
```
Comment (Mihndim2020) [inline README.md]: - [ ] Please you can delete this if the `Live demo`is not available, as this will make your readme file clean and professional.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @yusufnviiri, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean are organized code.
:heavy_check_mark: Your project is running free of `Linter errors`.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @yusufnviiri,

You've done well working on the concerns from the previous review 👏🏼.
Tasks for this project are now successfully implemented 💯.

### Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by khitermedachraf — COMMENTED** [review]
Hello @yusufnviiri 👋 you did Amazing work 🥇 .
- Clean Code 👍 .
- Github workflow 👍 .
- Professional README ✔️
Here are some best practices you can always implement to improve the quality of code because yours is great:

- Form fields have no labels, it would be good practice to add labels to your form
- It would be great if you used semantic tags
- Your commit messages should be more descriptive and precise
- Also it is best practice to begin committing messages with caps so kindly consider that in subsequent commits

**Pull request merged**